GetDKAN/dkan
# Admin updates

Improves menu organization and consolidates content admin into a single view, leaving "content" as is.

## QA Steps

1. Build a site with content.
2. Go to admin page, confirm that all DKAN content and config can be managed under the DKAN menu item
3. Confirm that the "Datasets" menu item provides all needed functionality to manage the catalog.
